RiotSharp/RiotApi.cs

```diff
@@ -334,123 +341,92 @@ public League GetMasterLeague(Region region, string queue)
                 region);
             return await Task.Factory.StartNew(() => JsonConvert.DeserializeObject<League>(json));
         }
+        #endregion
 
-        public MatchDetail GetMatch(Region region, long matchId, bool includeTimeline = false)
+        #region Match
+        public List<long> GetMatchIdsByTournamentCode(Region region, string tournamentCode)
         {
-            var json = requester.CreateGetRequest(
-                string.Format(MatchRootUrl, region.ToString()) + string.Format(IdUrl, matchId),
-                region,
-                includeTimeline
-                    ? new List<string> { $"includeTimeline={includeTimeline.ToString().ToLower()}" }
-                    : null);
-            return JsonConvert.DeserializeObject<MatchDetail>(json);
+            var json = requester.CreateGetRequest(MatchRootUrl +
+                                                  string.Format(MatchIdsByTournamentCodeUrl, tournamentCode), region);
+
+            return JsonConvert.DeserializeObject<List<long>>(json);
         }
-        #endregion
 
-        #region Match
-        public async Task<MatchDetail> GetMatchAsync(Region region, long matchId, bool includeTimeline = false)
+        public async Task<List<long>> GetMatchIdsByTournamentCodeAsync(Region region, string tournamentCode)
         {
-            var json = await requester.CreateGetRequestAsync(
-                string.Format(MatchRootUrl, region.ToString()) + string.Format(IdUrl, matchId),
-                region,
-                includeTimeline
-                    ? new List<string> {$"includeTimeline={includeTimeline.ToString().ToLower()}"}
-                    : null);
+            var json = await requester.CreateGetRequestAsync(MatchRootUrl +
+                                                             string.Format(MatchIdsByTournamentCodeUrl, tournamentCode),
+                region);
+
+            return await Task.Factory.StartNew(() => JsonConvert.DeserializeObject<List<long>>(json));
+        }
+
+        public Match GetMatch(Region region, long matchId, long? accountId = null)
```

Comment:
I think you forgot to remove the `accountId` here based on @JanOuborny 's comments